### PR TITLE
Service Cyborgs trade their shitty Violin for the epic Synthesizer.

### PR DESCRIPTION
**Pull request by zeroisthebiggay**
## About The Pull Request
Service Cyborgs got a synthesizer to play (not loud and obnoxious music) on a variety of instruments!
They also lost their Violin as it was mostly redundant as the Synthesizer can just use it's noise. They will however keep their Guitar for it's head bashing sound effects.

## Why It's Good For The Game
Gives the (second) most useless Cyborg caste something else to do other than play the same few songs because it's the only ones their instruments could play reasonably. Now you can even play the guitar and an instrument of your choice for sick combos or just take the synthesizer by itself and enjoy that sweet piano.

## Changelog
:cl: zeroisthebiggay
add: Gave the Service Cyborg a Synthesizer! Try not to destroy the Human populace's weak organic eardrums with it.
del: Service Cyborgs have lost their Violin to compensate for this. Your bashing in of heads with the Guitar will remain unimpeded however.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 1]
```diff
@@ -473,6 +473,7 @@
 		/obj/item/rsf,
 		/obj/item/instrument/violin,
```
Comment (Wilchenx) [inline code/modules/mob/living/silicon/robot/robot_modules.dm]: with piano synth added this and the guitar are reduntant now

Comment (zeroisthebiggay) [reply code/modules/mob/living/silicon/robot/robot_modules.dm]: maybe. but nothing will bring back the satisfaction of beating the captain to death with a guitar. maybe just the violin is a bit useless.

Comment (Wilchenx) [reply code/modules/mob/living/silicon/robot/robot_modules.dm]: oh fuck, you're absolutely correct

the guitar does make a unique sound upon hitting someone

Comment (zeroisthebiggay) [reply code/modules/mob/living/silicon/robot/robot_modules.dm]: yeah fuck the violin.

**Comment by zeroisthebiggay** [comment]
Righto so since this is my first ever PR and i'm inept and I can't figure out how to push my tree to the branch to remove violins i'm just going to label it out of scope for this pr _(unless i get baby step instructions on how the fuck i can do it but)_ but until then you can have the power of all three instruments at once and be **incredibly annoying**.

**Comment by zeroisthebiggay** [comment]
in attempt to do it i accidentally closed it
i have dumb

**Comment by AnturK** [comment]
Guitar seems bit redundant, if it's about unique hitsound just make cyborg synthesizer that uses that one. (Alternatively for bonus points let synthesizer choose it's own hitsound)

**Pull request merged**